Apps/WebClient/src/ClientApp/src/views/HomeView.vue

```diff
@@ -346,6 +363,11 @@ export default class HomeView extends Vue {
         this.$router.push({ path: "/covid19" });
     }
 
+    private handleClickOrganDonorCard(): void {
+        this.trackClickLink("organ_donor_card");
```

Comment:
This would generate an event with text "home_organ_donor_card".  This doesn't match the story criteria, but I think the criteria may be out of date. I'd recommend confirming with the product owner. (I'd also suggest "organ_donor_registration" instead of "organ_donor_card", just to match the behaviour with dataset quicklinks.)

Reply:
this is in line with the bc vaccine card click handler on line: [361](https://github.com/bcgov/healthgateway/pull/4832/files/ea2bef5c39691aab96145d0b581eab8411d71fb7#diff-667e82b527366fd44c255d3e5f34caf87c901d541e5cd5705d4d0e1c97b242b1R361)

I'll confirm with Nino none-the-less

Reply:
while compiling the question to Nino I took a look at the quicklinks and they too go through the trackClickLink method, thus this will also generate "home_organ_donor_registration".

So I'll update the text but its in line with all the other functionality